airbyte-webapp/src/config/index.ts

```diff
@@ -65,6 +68,9 @@ const config: Config = {
     baseUrl: "https://app.papercups.io",
     enableStorytime: window.PAPERCUPS_STORYTIME !== "disabled",
   },
+  openreplay: {
```

Comment:
do we need to do anything to keep this disabled while in dev mode?

Reply:
Added env variables for k8s here same as for Fullstory